package.json

```diff
@@ -42,6 +42,7 @@
 		"eslint": "^4.19.1",
 		"grunt": "^1.0.2",
 		"grunt-contrib-uglify": "^3.3.0",
+		"iconv-lite": "^0.4.24",
```

Comment:
Can we avoid adding a new dependency? As it's only used on test maybe we can use a simpler test